packages/esm-patient-attachments-app/src/attachments/attachments-grid-overview.component.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import styles from './attachments-grid-overview.scss';
+import { Attachment } from './attachments-types';
+import AttachmentThumbnail from './attachment-thumbnail.component';
+import { SkeletonPlaceholder } from 'carbon-components-react';
+
+interface AttachmentsGridOverviewProps {
+  isLoading: boolean;
+  attachments: Array<Attachment>;
+  deleteAttachment: (attachment: Attachment) => void;
+  onAttachmentSelect: (attachment: Attachment) => void;
+}
+
+const AttachmentsGridOverview: React.FC<AttachmentsGridOverviewProps> = ({
+  attachments,
+  isLoading,
+  onAttachmentSelect,
+}) => {
+  if (isLoading) {
+    return (
+      <div className={styles.galleryContainer}>
+        <SkeletonPlaceholder className={styles.attachmentThumbnailSkeleton} />
+        <SkeletonPlaceholder className={styles.attachmentThumbnailSkeleton} />
+        <SkeletonPlaceholder className={styles.attachmentThumbnailSkeleton} />
+        <SkeletonPlaceholder className={styles.attachmentThumbnailSkeleton} />
+      </div>
+    );
+  }
+
+  return (
+    <div className={styles.galleryContainer}>
+      {attachments.map((attachment, indx) => {
```

Comment:
5 letters for `index` was too expensive? 😂

Reply:
Gotta save those byts

Reply:
😆